### PR TITLE
Packages satysfi-fonts-charis-sil.1.0.0 and satysfi-fonts-charis-sil-doc.1.0.0

### PR DESCRIPTION
**Pull request by yuhr**
This pull-request concerns:
-`satysfi-fonts-charis-sil.1.0.0`: SATySFi Font Package for Charis SIL
-`satysfi-fonts-charis-sil-doc.1.0.0`: Document of SATySFi Font Package for Charis SIL

---
* Homepage: https://github.com/yuhr/satysfi-fonts-charis-sil
* Source repo: git+https://github.com/yuhr/satysfi-fonts-charis-sil.git
* Bug tracker: https://github.com/yuhr/satysfi-fonts-charis-sil/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

👇I don't understand what I am supposed to do for this. What do you mean by "choose"?

# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- [x] Add to snapshot `snapshot-develop` :: satysfi-fonts-charis-sil-doc.1.0.0 satysfi-fonts-charis-sil.1.0.0
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- [x] Add to snapshot `snapshot-stable-0-0-5` :: satysfi-fonts-charis-sil-doc.1.0.0 satysfi-fonts-charis-sil.1.0.0

### CODE REVIEW

**Comment by na4zagin3** [comment]
Let me try again.

File: packages/satysfi-fonts-charis-sil/satysfi-fonts-charis-sil.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+synopsis: "SATySFi Font Package for Charis SIL"
+description: """
+SATySFi Font Package for Charis SIL
+
+This package installs fonts from https://software.sil.org/charis/.
+"""
+maintainer: "Yu Shimura <mail@yuhr.org>"
+authors: "Yu Shimura <mail@yuhr.org>"
+license: "MIT"
```
Comment (na4zagin3) [inline packages/satysfi-fonts-charis-sil/satysfi-fonts-charis-sil.1.0.0/opam:10]: Can you update the license?

```suggestion
license: "OFL-1.0"
```

File: packages/satysfi-fonts-charis-sil-doc/satysfi-fonts-charis-sil-doc.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+   "--name" "fonts-charis-sil-doc"
+   "--prefix" "%{prefix}%"
+   "--script" "%{build}%/Satyristes"]
+]
+install: [
+  ["satyrographos" "opam" "install"
+   "-name" "fonts-charis-sil-doc"
+   "-prefix" "%{prefix}%"
+   "-script" "%{build}%/Satyristes"]
+]
```
Comment (na4zagin3) [inline packages/satysfi-fonts-charis-sil-doc/satysfi-fonts-charis-sil-doc.1.0.0/opam:31]:
```suggestion
]
url {
  src:
    "https://github.com/yuhr/satysfi-fonts-charis-sil/archive/v1.0.0.tar.gz"
  checksum: [
    "md5=738e8783abe0522fc4f67dbd1b7045dc"
    "sha512=bc56c9a07d86d9586da1026434866665e8c9957d4619937011123b362fdce643665ca21bed3cc6fbed14a131023977b140de12af5beb2bbf53d29a15ff37329d"
  ]
}
```

File: packages/satysfi-fonts-charis-sil/satysfi-fonts-charis-sil.1.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+]
+build: [
+  ["unzip" "-j" "-o" "CharisSIL-5.000.zip" "*.ttf" "-d" "charis-sil"]
+]
+install: [
+  ["satyrographos" "opam" "install"
+   "--name" "fonts-charis-sil"
+   "--prefix" "%{prefix}%"
+   "--script" "%{build}%/Satyristes"]
+]
```
Comment (na4zagin3) [inline packages/satysfi-fonts-charis-sil/satysfi-fonts-charis-sil.1.0.0/opam:33]:
```suggestion
]
url {
  src:
    "https://github.com/yuhr/satysfi-fonts-charis-sil/archive/v1.0.0.tar.gz"
  checksum: [
    "md5=738e8783abe0522fc4f67dbd1b7045dc"
    "sha512=bc56c9a07d86d9586da1026434866665e8c9957d4619937011123b362fdce643665ca21bed3cc6fbed14a131023977b140de12af5beb2bbf53d29a15ff37329d"
  ]
}
```

**Review by na4zagin3 — CHANGES_REQUESTED** [review]

**Review by na4zagin3 — APPROVED** [review]

**Comment by na4zagin3** [comment]
@yuhr now it's ready to choose the follow-up actions.

**Comment by na4zagin3** [comment]
I'm merging this package.  Thanks!

**Comment by yuhr** [comment]
@na4zagin3 Thanks, and sorry for being late 😭 By the way, I definitely recommend using more clear words for the automation instructions, for example "wait until CI passes", and "tick checkboxes" instead of "choose".

**Comment by na4zagin3** [comment]
It makes sense.  I'll address them.  Thanks!
